editor/editor_properties.cpp

```diff
@@ -1321,6 +1321,19 @@ void EditorPropertyInteger::_value_changed(int64_t val) {
 
 void EditorPropertyInteger::update_property() {
 	int64_t val = get_edited_property_value();
+	if (is_checkable() && !is_checked()) {
+		Control *control = Object::cast_to<Control>(get_edited_object());
+		if (control) {
+			String property_name = get_edited_property();
+			if (property_name.begins_with("theme_override_font_sizes/")) {
+				String name = property_name.get_slicec('/', 1);
```

Comment:
I'd just strip the length of the prefix here, as we know the position of the slash

Reply:
Yes, that would be better. Thanks. There are similar codes in `Control` and `Window`.